modules/lb-vshn-hiera/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "router_ip_addresses" {
+  type        = list(string)
+  description = "Private IPV4 addresses of nodes running ingress routers"
+}
+
+variable "bootstrap_node" {
+  type        = string
+  description = "The bootstrap nodes private IPV4 adsress"
+  default     = ""
+}
+
+variable "node_name_suffix" {
+  type        = string
+  description = "Suffix to use for node names"
+}
+
+variable "cluster_id" {
+  type        = string
+  description = "ID of the cluster"
+}
+
+variable "privnet_cidr" {
+  default     = "172.18.200.0/24"
+  description = "CIDR of the private net to use"
+}
+
+variable "lb_names" {
+  description = "The hostnames of the loadbalancers"
+  type        = list(string)
+}
+
+variable "lb_cloudscale_api_secret" {
+  type        = string
+  description = "Read-Only API secret to access floating ips"
+}
+
+variable "hieradata_repo_user" {
+  type        = string
+  description = "User used to check out the hieradata git repo"
+}
+
+variable "api_vip_network" {
+  type        = string
+  description = "Floating IP for the API"
```

Comment:
@simu I don't yet completely understand what the different floating IPs are for. Could you help me improve the descriptions?

Reply:
```suggestion
  description = "Floating IP for the Kubernetes/OpenShift API"
```